### PR TITLE
Fix gallery

### PR DESCRIPTION
**Pull request by drbenvincent**
Further to #353, this PR removes left over headings in the gallery.

The single BART and Spline examples were put into Case Studies to avoid a gallery rendering error, but I forgot to remove the sections from the `gallery.rst`.

So this PR fixes that and should enable #353 to be closed.

### CODE REVIEW

**Comment by drbenvincent** [comment]
Working correctly EXCEPT FOR no gallery thumbnails under How To right at the bottom

File: examples/gallery.rst [hunk 1]
```diff
@@ -124,16 +114,4 @@ How to
 .. nbgallery::
     :name: howto
     :glob:
-    :reversed:
-
-    howto/*
```
Comment (OriolAbril) [inline examples/gallery.rst:129]: This doesn't look right and the how tos are gone in the preview: https://pymcio--361.org.readthedocs.build/projects/examples/en/361/gallery.html#how-to

Comment (drbenvincent) [reply examples/gallery.rst:129]: Just noticed lack of thumbnails. What is wrong with it?

Comment (OriolAbril) [reply examples/gallery.rst:129]: I think you deleted the `howto/*` with the latest merge from main somehow and so nothing is listed inside that "nbgallery"

Comment (drbenvincent) [reply examples/gallery.rst:129]: Ok. Will come at this again tomorrow with a fresh head.

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**